### ISSUE TITLE
Editing old flights causes duration error

### ISSUE THREAD

**Issue by nirenjan** [johanohly/AirTrail#105]
### The bug

I have some old flights that I had imported from Jetlog. When I try to edit the flight (update seats, correct airline, etc.), I get an error "Arrival must be after departure". I tested creating a dummy flight with the same dates and times, and I was able to create the flight without any issue, however, attempting to edit that same flight gave me errors again.

Editing imported flight causes issues
![image](https://github.com/user-attachments/assets/44ba71fd-832e-45fb-800b-82d93f1ce35c)

Adding flight has no problem
![image](https://github.com/user-attachments/assets/2f876d9e-cbc8-410c-b6c0-989243ca0582)

Editing test flight shows issues
![image](https://github.com/user-attachments/assets/73249307-01e0-4749-874c-8c2981be403c)

### Operating System that is running AirTrail

Ubuntu

### AirTrail Version

v1.0.1

### Your docker-compose.yml content

```YAML
services:
  db:
    container_name: airtrail_db
    image: postgres:16-alpine
    restart: always
    env_file:
      - .env
    environment:
      POSTGRES_DB: ${DB_DATABASE_NAME}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_PASSWORD: ${DB_PASSWORD}
    volumes:
      - db_data:/var/lib/postgresql/data
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U ${DB_USERNAME} -d ${DB_DATABASE_NAME}"]
      interval: 5s
      timeout: 5s
      retries: 5

  airtrail:
    container_name: airtrail
    image: johly/airtrail:latest
    restart: always
    env_file:
      - .env
    ports:
      - 1380:3000
    depends_on:
      db:
        condition: service_healthy

volumes:
  db_data:
```

### Reproduction steps

1. Create or import flight (KSFO->KEWR)
2. Attempt to edit said flight
3. Observe error

### Additional information

I've also seen this error in other flights, mainly in east bound flights where the arrival time is usually in the early hours, and the departing flight leaves later in the day.

**Comment by johanohly** [comment]
Thank you for reporting this and providing such detailed reproduction steps. 

Noting that this only occurs when editing an existing flight made it straightforward to identify the issue, and it will be fixed in the next release.

**Comment by johanohly** [comment]
This should be fixed in v1.1.0. Do let me know how it goes.

**Comment by nirenjan** [comment]
Looks great - the editing works as expected.
